FakeTrello/Models/Card.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Web;
+
+namespace FakeTrello.Models
+{
+    public class Card
+    {
+        [Key]
+        public int CardId { get; set; }
+
+        public string Title { get; set; }
+
+        public string Description { get; set; }
+        
+        // List of tasks in a checklist
+        public List Checklist { get; set; }
```

Comment:
This is interesting! I didn't think of it this way. You might have some trouble with many many nested cards and lists, but I like the way you are thinking outside the box here.